src/StringParser/Yaml.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Noodlehaus\StringParser;
+
+use Exception;
+use Symfony\Component\Yaml\Yaml as YamlParser;
+use Noodlehaus\Exception\ParseException;
+
+/**
+ * YAML string parser
+ *
+ * @package    Config
+ * @author     Jesus A. Domingo <jesus.domingo@gmail.com>
```

Comment:
Please, add yourself as author of this class, removing the others.

Reply:
I've fixed this.